restore-hooks_product-requirements.md

```diff
@@ -0,0 +1,337 @@
+
+**<span style="text-decoration:underline;">USE CASE 6</span>**
+
+
+    **Title: **Specify backup snapshot to use for restore
+
+
+    **Description: **As a user, I would like the option to specify a specific backup snapshot for use by Velero during restore create. I would like to do this instead of using the default most recent backup restore snapshot.
+
+**<span style="text-decoration:underline;">______________________________________________________________</span>**
+
+**<span style="text-decoration:underline;">USE CASE 7</span>**
+
+
+    **Title: **Restore of a stateful application (Unquiescing data from a quiesced backup)
+
+
+    **Description: **As a user, I would like to prepare a backup to be restored without having to freeze an in-use volume/volume that an application is using.
+
+
+    As a user, I would like to unquiesce data during a restore to prevent the need to shut down the database and disrupt the application end user experience. 
+
+
+    _Note: Some dbs have built in way to back up _
+
+**<span style="text-decoration:underline;">______________________________________________________________</span>**
+
+**<span style="text-decoration:underline;">USE CASE 8</span>**
+
+
+    **Title: **Display/surface restore status 
+
+
+    **Description:** As a user, I would like to see the status of my restore status surfaced from the pod volume restore status. Including statuses:  Pending, Running/In Progress, Succeeded, Failed, Unknown.
+
+**<span style="text-decoration:underline;">______________________________________________________________</span>**
+
+**<span style="text-decoration:underline;">USE CASE 9</span>**
+
+
+    **Title: **Restore metadata
+
+
+    **Description: **As a user, I would like to have the metadata of the contents of what was restored using Velero.
+
+
+    _Note: Kubernetes patterns may result in some snapshot metadata being overwritten during restore operations._
+
+**<span style="text-decoration:underline;">______________________________________________________________</span>**
+
+**<span style="text-decoration:underline;">USE CASE 10</span>*
+
+    **Title: **Increase default restore and retry limits. 
+
+
+    **Description: **As a user, I would like to increase the default restore retry and timeout limits from the default values to some other value I would like to specify. 
+
+
+    _Note: See use case 11 for the default value specifications. _
+
+**<span style="text-decoration:underline;">______________________________________________________________</span>**
+
+User Experience
+
+The following is representative of what the user experience could look like for Velero restore pre-hooks and post-hooks. 
+
+**_Note: These examples are representative and are not to be considered for us in pre- and post- restore hook operations until the technical design is complete._**
+
+**Restore Pre-Hooks**
+
+<span style="text-decoration:underline;">Container Command</span>
```

Comment:
It isn't clear to me what this section is for. Can you please clarify?

Reply:
Product requirements attempt to provide examples of what the user experience in the command line might look like. Similar to a wireframe for a UI design. As such, this is not the actual design or implementation of the feature - just a high-level concept of what it might look like (just in the same way a wireframe i that and is not considered final or actual experience until the UI/UX design is done and the implementation is complete).